1. Провести дисперсионный анализ для определения того, есть ли различия среднего роста среди взрослых футболистов, хоккеистов и штангистов. Даны значения роста в трех группах случайно выбранных спортсменов: Футболисты: 173, 175, 180, 178, 177, 185, 183, 182. Хоккеисты: 177, 179, 180, 188, 177, 172, 171, 184, 180. Штангисты: 172, 173, 169, 177, 166, 180, 178, 177, 172, 166, 170.

In [3]:
import numpy as np
from scipy import stats

In [5]:
fb = np.array([173, 175, 180, 178, 177, 185, 183, 182])
hc = np.array([177, 179, 180, 188, 177, 172, 171, 184, 180])
bb = np.array([172, 173, 169, 177, 166, 180, 178, 177, 172, 166, 170])
n = len(fb) + len(hc) + len(bb)

28

In [4]:
stats.f_oneway(fb, hc, bb)

F_onewayResult(statistic=5.500053450812596, pvalue=0.010482206918698694)

5.5 > 3.83 (table), p-val < 0.05 => статистически значимая разница в росте есть с уровнем значимости 0.05